File: modul6/C/PRAK602_2310817210017_MUHAMMADNURHUDABAIHAQI.c
#include <stdio.h>

int main() {
    int jumlah_ruangan, a;
    scanf("%d", &jumlah_ruangan);

    int zetsu_putih[jumlah_ruangan];
    for (a = 0; a < jumlah_ruangan; a++) {
        scanf("%d", &zetsu_putih[a]);
    }
    for (a = 0; a < jumlah_ruangan; a++) {
        int jumlah_setelah_pembelahan = zetsu_putih[a] * (a + 1);
        printf("%d ", jumlah_setelah_pembelahan);
    }
    printf("\n");
    return 0;
}

File: modul6/PYTHON/PRAK602_2310817210017_MUHAMMADNURHUDABAIHAQI.py
jumlah_ruangan = int(input())
zetsu_putih = list(map(int, input().split()))

for a in range(jumlah_ruangan):
    jumlah_setelah_pembelahan = zetsu_putih[a]* (a + 1)
    print(jumlah_setelah_pembelahan, end=" ")

print()
